src/relay/backend/contrib/cmsisnn/tir_to_runtime.cc

```diff
@@ -88,8 +88,12 @@ class CodeGenCMSISNN : public codegen::CodeGenCHost {
   std::string EmitCMSISNNConvParams(std::ostream& os, int32_t input_offset, int32_t output_offset,
                                     int32_t stride_w, int32_t stride_h, int32_t padding_w,
                                     int32_t padding_h, int32_t dilation_w, int32_t dilation_h,
-                                    int32_t clip_min, int32_t clip_max) {
-    std::string struct_name = "conv_params";
+                                    int32_t clip_min, int32_t clip_max, int32_t depth_multiplier) {
+    std::string struct_name = "cmsis_nn_conv_params";
+    std::string instance_name = "conv_params";
+    if (depth_multiplier != -1) {
+      struct_name = "cmsis_nn_dw_conv_params";
+    }
```

Comment:
Minor suggestion, keep manipulation of the same variable close together:
```suggestion
    std::string instance_name = "conv_params";
    std::string struct_name = "cmsis_nn_conv_params";
    if (depth_multiplier != -1) {
      struct_name = "cmsis_nn_dw_conv_params";
    }
```

Reply:
ACK